src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+header {
+  width: 100%;
+  flex-shrink: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 10px 50px;
+}
+
+.header__logo {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  gap: 20px;
```

Comment:
use margins instead, now your work doesn't look according to the design

Reply:
Not fixed